### ISSUE TITLE
Rename repo

### ISSUE THREAD

**Issue by abalmos** [OADA/oada-client-secret#7]
Now we are using standard jwt-bearer client assertions we might consider renaming the repo and npm project.

Maybe jwt-bearer-client-assertion or just jwt-client-assertion ?

**Comment by abalmos** [comment]
Or to stay consistent:

jwt-bearer-client-assertion-js
or
jwt-client-assertion-js

**Comment by awlayton** [comment]
This does  more than just what is in the jwt-bearer spec.

**Comment by abalmos** [comment]
I don't think it will after we move to dyn-client-reg but let's re-evaluate then
